frontend/src/views/SQR/Form.vue

```diff
@@ -64,9 +64,14 @@
                 <v-btn v-if="!isCompleted" @click="saveDraft">
                   Save draft
                 </v-btn>
-                <v-btn type="submit" color="primary" class=" mx-2"
-                  >Submit</v-btn
+                <v-btn
+                  v-if="$can('update', 'SQR/Form')"
```

Comment:
Here semantically the action is `submit`, no?

Reply:
Moreover, anyone can submit if it's not Done, but only particular roles can submit Done ones. I guess this conditional rule could be handled by CASL.

Reply:
I was using the default actions but I will switch to better named actions. I think putting this conditional rule in CASL would be difficult because data can't be easily passed into the ability. I think `!isCompleted || $can('submit', 'SQR/Form')` in the v-if might be better.

Reply:
Agreed.